deploy/csi-hostpath-plugin.yaml

```diff
@@ -60,8 +60,8 @@ spec:
               name: csi-data-dir
 
         - name: hostpath
-          # precise version of the 4.8.0 tag used for the blog post branch/tag
-          image: quay.io/openshift/origin-csi-driver-projected-resource@sha256:2007468d111cdb9bd313087c8cf267d584c4bd7489f558f4a29eb3466325b85c
+          # precise version of the 4.8.0 tag after commit 0478e08051006ade5254e179a5673ed95302b2ef (employing this level of precision for blog post)
+          image: quay.io/openshift/origin-csi-driver-projected-resource@sha256:52eeacd4480a4cea2a0e5a7230d9ce77deead91db823910971743a0482d5f24f
```

Comment:
unless more bugs are found, this version of the image could be the one used for the blog post

subsequent commits around README tweaks, pulling in of OLM install, cutting of blog post tag on repo would in theory not update the core driver image

but as OLM install comes in, still TBD on whether it uses this deployment def, or a separate one